### ISSUE TITLE
How can I stop monitor after first element visibility?

### ISSUE THREAD

**Issue by codev0** [vissense/vissense#32]
In my case I have piechart plugin. And show it when visible. Its works.
But plugin initialized each time seen.
```
if($("div").is(".statistic-block")) {
    var $statsBlock = $(".statistic-block");

    var $statsBlockVisibility = VisSense($statsBlock[0]).monitor({
      visible: function() {
        $('#myStat1').circliful();
        $('#myStat').circliful();
        $('#myStat3').circliful();
        $('#myStat2').circliful();

        $statsBlock.addClass("visible");
      }
    }).start();

    if($statsBlock.hasClass("visible")) {
      console.log("stop");
      $statsBlockVisibility.stop();
    }
  }
```

**Comment by theborakompanioni** [comment]
@codev0 You can stop the monitor as soon as the element is visible. All hooks receive the monitor object as first argument, so you can stop it like this:
```
VisSense($statsBlock[0]).monitor({
  visible: function(monitor) {
    $('#myStat1').circliful();
    $('#myStat').circliful();
    $('#myStat3').circliful();
    $('#myStat2').circliful();

    monitor.stop();
  }
}).start();
```

**Comment by codev0** [comment]
@theborakompanioni Hi. Thanks.
Its work.
But if the video block is visible right after the page is loaded, then it is executed again every time the block enters viewport

**Comment by theborakompanioni** [comment]
@codev0 Can you provide some code so I can reproduce the problem?

**Comment by theborakompanioni** [comment]
@codev0 I played around with the [video demo](https://jsbin.com/maqaco/edit?js,output) but could not reproduce your problem. Can you provide further information?

**Comment by codev0** [comment]
@theborakompanioni 
https://output.jsbin.com/hixijujecu

**Comment by theborakompanioni** [comment]
@codev0 Ah, ok. I could reproduce the problem - thanks for reporting. I'll take a look at it this week and keep you updated. Thanks!

**Comment by theborakompanioni** [comment]
@codev0 Thanks for your effort. Has been fixed with ca93cab9. [v0.10.0](https://github.com/vissense/vissense/releases/tag/0.10.0) has been released. This behavior should have been covered by tests long ago!
